packages/eslint-plugin/src/util/index.ts

```diff
@@ -12,7 +12,6 @@ export * from './nullThrows';
 export * from './objectIterators';
 export * from './propertyTypes';
 export * from './requiresQuoting';
-export * from './types';
 
```

Comment:
one thing I would suggest here - could you please export everything from the utils package here?

That way we don't have to import utils from two different places in our plugin and can continue to just import from our `../util` file.

It's going to be a bit of a pain for you to undo everything (sorry!!), but it makes the code easier for everyone in future and it will prevent people resolving merge conflicts on existing PRs.

```ts
export * from '@typescript-eslint/experimental-utils/dist/eslint-utils/types';
```